File: 2019-01-02/2581/2581.c
#include <stdio.h>
#include <math.h>

int sum = 0;
int min = 10002;

int estoras(int number);
void setMin(int M, int N);

int main(){
	int M, N;
	scanf("%d %d", &M, &N);

	setMin(M, N);
	
	if(sum == 0){
		printf("-1");
	}else{
		printf("%d\n%d", sum, min);
	}
	
}

void setMin(int M, int N){
	for(int i = M; i <= N; i++){
		if( i != 1){
			if(estoras(i)){
				if(min > i)
				min = i;
			}
		}
	}
}

int estoras(int number){
	for(int i = 2; i <= sqrt(number); i++){
		if(number % i == 0){
			return 0;
		}
	}
	sum += number;
	return 1;
}

File: 2019-01-02/2581/2581.java
import java .io.*;


class Main{
	static int sum = 0;
	static int min = 10001;

	public static void main(String[] args) throws Exception{
		BufferedReader bfr = new BufferedReader(new InputStreamReader(System.in));
		
		int M = Integer.parseInt(bfr.readLine());
		int N = Integer.parseInt(bfr.readLine());
		for(int i = M; i <= N; i++){
			if(i != 1){
				if(estoras(i)){
					if(min > i)
						min = i;
				}
			}
		}

		printResult();
	}

	private static void printResult(){
		if(sum == 0)
			System.out.println("-1");
		else 
			System.out.printf("%d\n%d", sum, min);
	}

	private static boolean estoras(int number){
		for(int i = 2; i <= Math.sqrt(number); i++){
			if(number % i == 0){
				return false;
			}
		}
		sum += number;
		return true;
	}
	
}